packages/jest-phabricator-coverage/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "jest-phabricator-coverage",
```

Comment:
didn't we decide to call this `jest-phabricator`? That's the name I reserved on npm a couple of weeks back.